compiler/src/dotty/tools/dotc/printing/SyntaxHighlighting.scala

```diff
@@ -48,6 +49,11 @@ object SyntaxHighlighting {
           highlightRange(pos.start, pos.end, color)
       }
 
+      def highlightComments(commentsPos: mutable.MutableList[Position]): Unit = {
```

Comment:
This is used only once. I would inline at call site

Reply:
Do you mean to say use `@inline` or replace call invocation with logic?

Reply:
replace call invocation with logic